### DISCUSSION TITLE
Limit Up/Download Speed

### DISCUSSION THREAD

**Discussion by Dlay2017** [filebrowser/filebrowser#2594] [Ideas]
Possibility to limit Up/Download Speed

Please consider an Option to do so. In Default, FileBrowser uses full Bandwith and that stops other network applications from performing well, as soon as a download starts

**Comment by iReisen** [comment]
Yes, this is very important. my browser actually crashes regularely if i upload a folder with a bunch of items at once, which also kills the connection of other programs.

**Comment by assassinliujie** [comment]
maybe use nginx to do that
